pandasai/agent/base.py

```diff
@@ -49,6 +58,8 @@ def __init__(
         config: Optional[Union[Config, dict]] = None,
         memory_size: Optional[int] = 10,
         pipeline: Optional[Type[GenerateChatPipeline]] = None,
```

Comment:
@milind-sinaptik  I think we should either remove pipeline or remove the other two and keep only one pipeline.

Reply:
done